packages/math-input/src/components/tabbar/tabbar.tsx

```diff
@@ -32,16 +33,41 @@ type Props = {
 function Tabbar(props: Props): React.ReactElement {
     const {items, onClickClose, selectedItem, onSelectItem, style} = props;
 
+    const selectedIndex = items.findIndex((item: KeypadPageType) => {
+        return item === selectedItem;
+    });
+
+    const [focus, setFocus] = useState(
+        selectedIndex === -1 ? 0 : selectedIndex,
+    );
+
+    // Custom function to handle arrow key navigation for the TabBar for each TabItem.
+    const useArrowKeyFocus = (e) => {
+        if (e.keyCode === 39) {
+            // Right arrow
+            setFocus(focus === items.length - 1 ? items.length - 1 : focus + 1);
```

Comment:
Just curious: Do we not want the focus to cycle to the other side of the tabbar when it reaches one of the ends? I guess I was expecting it to act like a menu where after reaching the end, the next arrow would put focus back on the first item. Maybe that's just me.

Reply:
I guess maybe for a tablist, this is the correct behavior. Just a curiousity.

Reply:
For me it felt like it would be more confusing if it cycled back, but I'm good with either implementation. Let me ask Caitlyn real quick.

Reply:
So, it looks like cycling back to the other end is a proper (though optional) behavior: [ARIA Tabs Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)

I also noticed [ARIA Toolbar Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/), and wondered if that might be more appropriate here (for `role` definition). Same expectations for arrow key navigation. I'm not saying that this needs changing, just pointing out another option, in case it helps.